<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maniphest Resource Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="authBox">
         <property name="title">
          <string>Authentication</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="phabricatorUrlLabel">
            <property name="text">
             <string>Phabricator &amp;URL:</string>
            </property>
            <property name="buddy">
             <cstring>phabricatorUrlEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="phabricatorUrlEdit">
            <property name="text">
             <string>https://</string>
            </property>
            <property name="placeholderText">
             <string>https://</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="apiTokenLabel">
            <property name="text">
             <string>Conduit API &amp;Token:</string>
            </property>
            <property name="buddy">
             <cstring>apiTokenEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="apiTokenEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="howToGetTokenHelp">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;To obtain the Conduit API Token, open your account settings on the Phabricator instance in web browser and select&lt;span style=&quot; font-style:italic;&quot;&gt;Conduit API Tokens&lt;/span&gt; tab in the left menu. Click &lt;span style=&quot; font-style:italic;&quot;&gt;Generate API Token,&lt;/span&gt; confirm creation of a new token and then copy-paste the token here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="toolTipDuration">
             <number>0</number>
            </property>
            <property name="text">
             <string>&lt;a href=&quot;#&quot;&gt;How to obtain the API token?&lt;/a&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="syncGroupBox">
         <property name="title">
          <string>Synchronization</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="refreshCheckBox">
            <property name="text">
             <string>Automatically refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="refreshIntervalLabel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Refresh every</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="refreshInterval">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string> minutes</string>
              </property>
              <property name="minimum">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="maniphestTab">
      <attribute name="title">
       <string>Maniphest</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="maniphestProjectsLabel">
         <property name="text">
          <string>Projects to synchronize:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="maniphestProjectsView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="maniphestProgressBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="maniphestRefreshButton">
         <property name="text">
          <string>&amp;Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
